### DISCUSSION TITLE
do previous remark-stringify versions have options.resourceLink?

### DISCUSSION THREAD

**Discussion by stefanprobst** [remarkjs/remark#755] [Q&A] [answered]
with the current `remark-stringify` version it is possible to avoid saving links as autolinks with the [`resourceLink: true`](https://github.com/syntax-tree/mdast-util-to-markdown#optionsresourcelink) option.

does a similar option exist for previous remark versions as well? i could not find it in [this list](https://github.com/remarkjs/remark/tree/ca5a9853971fa8cb26b1b21a3ba22b882d05c9ec/packages/remark-stringify#options).

---

for context: the markdown widget in netlify-cms uses an older version of remark, and i'm trying to teach it to not trying to stringify links as autolinks if possible, so there's one less issue with rendering the content with xdm.

**Comment by wooorm** [comment, marked as answer]
Hmm nope, indeed, it wasn’t there before.

You could create a custom plugin to define a “visitor”, which was the way to change how something serializes before:

```
this.Compiler.prototype.visitors.link = ...
```

And then something like https://github.com/remarkjs/remark/blob/remark-stringify%408.1.1/packages/remark-stringify/lib/visitors/link.js but w/o the first `if` branch.

Or maybe use https://www.npmjs.com/package/patch-package to change the old parser?

**Reply by stefanprobst** [reply]
thanks! i'll try with `patch-package`
